smart-contracts/contracts/cl-vault/Cargo.toml

```diff
@@ -45,27 +45,27 @@ cosmwasm-schema = "1.1.3"
 cosmwasm-std = "1.1.3"
 cosmwasm-storage = "1.1.3"
 cw-controllers = "1.0.1"
-cw-dex = "0.1.1"
+cw-dex = "0.4.0"
 cw-dex-router = {version = "0.1.0", features = ["library"]}
 cw-storage-plus = "1.0.1"
-cw-utils = "0.16.0"
+cw-utils = "1.0.2"
 cw-vault-multi-standard = {git = "https://github.com/quasar-finance/cw-vault-standard", features = ["lockup", "force-unlock"]}
 cw-vault-token = "0.1.0"
 cw2 = "1.0.1"
 cw20 = "1.0.1"
 cw20-base = "1.0.1"
-derive_builder = "0.11.2"
-liquidity-helper = "0.1.0"
-num_enum = "0.6.1"
-osmosis-std = "0.17.0-rc0"
+derive_builder = "0.12.0"
+liquidity-helper = "0.2.1"
+num_enum = "0.7.0"
+osmosis-std = "0.19.2"
 prost = {version = "0.11", default-features = false}
 schemars = "0.8.10"
 semver = "1"
 serde = {version = "1.0.145", default-features = false, features = ["derive"]}
 thiserror = {version = "1.0.31"}
 
 [dev-dependencies]
-cw-multi-test = "0.16.2"
-osmosis-test-tube = {git = 'https://github.com/osmosis-labs/test-tube.git', version = "17.0.0-rc0"}
+cw-multi-test = "0.17.0"
+osmosis-test-tube = {git = 'https://github.com/osmosis-labs/test-tube.git', version = "19.0.0"}
```

Comment:
This was already bumped in https://github.com/quasar-finance/quasar/pull/512, this PR should reflect that change too